app/decorators/swagger/route_decorator.rb

```diff
@@ -67,4 +68,39 @@ def security
 
     [{security_scheme.key => []}]
   end
+
+  def x_amazon_apigateway_integration(api_gateway_integration)
+    return unless api_gateway_integration
+
+    parameters = path_parameters
+
+    cache_key_parameters = parameters.map do |parameter|
+      'integration.request.path.' + parameter
+    end
+
+    request_parameters = {}
+    parameters.each do |parameter|
+      request_parameters['integration.request.path.' + parameter] = 'method.request.path.' + parameter
+    end
+
+    {
+      cacheKeyParameters: cache_key_parameters,
```

Comment:
From what I understand cache_key_parameters are keys of request_parameters, so you can do `{ cacheKeyParameters: request_parameters.key, ...`